utils/SendTransactionsUseCase.tsx

```diff
@@ -0,0 +1,38 @@
+import {
+  Connection,
+  VersionedTransaction,
+  TransactionSignature,
+  SendOptions,
+} from '@solana/web3.js';
+import {decode} from 'bs58';
+
+export class SendTransactionsUseCase {
+  static readonly SIGNATURE_LEN = 64;
+  static readonly PUBLIC_KEY_LEN = 32;
+
+  static async sendSignedTransactions(
+    signedTransactions: Array<Uint8Array>,
+    minContextSlot: number | undefined,
+  ): Promise<Uint8Array[]> {
+    const connection = new Connection(
+      'https://api.devnet.solana.com',
+      'confirmed',
+    );
+    const signatures: Uint8Array[] = await Promise.all(
+      signedTransactions.map(async byteArray => {
+        const transaction: VersionedTransaction =
+          VersionedTransaction.deserialize(byteArray);
+
+        const sendOptions: SendOptions = {
+          minContextSlot: minContextSlot,
+        };
+        const signature: TransactionSignature =
+          await connection.sendTransaction(transaction, sendOptions); // here
```

Comment:
4. Getting an "Unhandled promise" rejection here with `TypeError: undefined is not a function`
[Stack Trace](https://gist.github.com/Michaelsulistio/5f498973d3e4a11b725b93d6b0da9d5b)